src/qt/bitcoingui.cpp

```diff
@@ -402,6 +402,7 @@ void BitcoinGUI::createActions()
         });
         connect(m_close_wallet_action, &QAction::triggered, [this] {
             m_wallet_controller->closeWallet(walletFrame->currentWalletModel(), this);
+            labelWalletHDStatusIcon->hide();
```

Comment:
This in the wrong place, should be in `BitcoinGUI::removeWallet` - then it will also work if you open the console and call `unloadwallet`.

Reply:
Ooops, many thanks! Will update.